src/Monolog/Handler/LogglyHandler.php

```diff
@@ -43,12 +44,16 @@ public function __construct($token, $level = Logger::DEBUG, $bubble = true)
 
     public function setTag($tag)
     {
-        $this->tag = $tag;
+        $tag = !empty($tag) ? $tag : array();
+        $this->tags = is_array($tag) ? $tag : array($tag);
     }
 
     public function addTag($tag)
     {
-        $this->tag = (strlen($this->tag) > 0) ? $this->tag .','. $tag : $tag;
+        if (!empty($tag)) {
+            $tag = is_array($tag) ? $tag : array($tag);
+            $this->tags = array_merge($this->tags, $tag);
```

Comment:
@Seldaek you're right. i forgot this is a list not a dictionary. Ill update it to add array_unique